app/src/main/java/com/github/mik629/android/fundamentals/domain/model/Movie.kt

```diff
@@ -0,0 +1,13 @@
+package com.github.mik629.android.fundamentals.domain.model
+
+data class Movie(
```

Comment:
maybe vice versa?
details should contains more

Reply:
Yeah, also included in the issue